src/components/WarningModal/WarningModal.js

```diff
@@ -28,12 +28,14 @@ class WarningModal extends React.PureComponent {
   };
 
   componentDidUpdate(prevProps) {
-    if (!prevProps.isOpen && this.props.isOpen) {
+    const { isOpen, closeElements } = this.props;
+
+    if (!prevProps.isOpen && isOpen) {
       // TODO want to do make something like'closeOtherElements', but that for latter
-      const popUpToClose = getPopupElements().filter(
-        ele => ele !== 'warningModal',
+      const popupElementsToClose = getPopupElements().filter(
+        ele => ele !== 'warningModal' && ele !== 'leftPanel',
```

Comment:
Do you think it's better if just we just remove `leftPanel` in `getPopupElements`? Also warningModal isn't in the popup elements so we don't need to filter it out I think?

Reply:
Right now it seems the `warningModal` is the only thing that is using `getPopupElements` so I think now would be a good time to discuss how it should be used. 

Lot of compounds are just hard coding stuff to close like MenuOverlay.js
https://github.com/PDFTron/webviewer-ui/blob/7c1be32d0b448980e1785cb6d45c73bd97f86933/src/components/MenuOverlay/MenuOverlay.js#L47

and SearchOverlay.js
https://github.com/PDFTron/webviewer-ui/blob/7c1be32d0b448980e1785cb6d45c73bd97f86933/src/components/SearchOverlay/SearchOverlay.js#L85

The idea was to have a list of all "popup" somewhere so if a new one was added we didn't need to update 5+ components. I will remove "leftPanel" from the list because I don't think we want to close it in most cases.

Also after this PR, it would be a good time to have other components start using the `getPopupElements` helper (or I should remove it if it doesn't seem helpful or refactor it into a "closeOtherPopupElement" helper)